packages/react/src/components/form/form.test.tsx

```diff
@@ -75,10 +75,11 @@ describe("#Form", () => {
   test("should show the success state on successful login", async () => {
     const logInMock = vi.fn(() => Promise.resolve(TEST_USER));
     const user = userEvent.setup();
+    const onSuccess = vi.fn();
 
     render(
       <TestSlashIDProvider sdkState="ready" logIn={logInMock}>
-        <Form />
+        <Form onSuccess={onSuccess} />
```

Comment:
Would you mind adding an additional test case for this? IMO it adds value to have tests as living documentation. So we can have something like `test("should call the onSuccess callback if provided on a successful login", () => ...)`